src/utils/zif_abapgit_log.intf.abap

```diff
@@ -1,21 +1,62 @@
 INTERFACE zif_abapgit_log
   PUBLIC .
 
+  TYPES:
+    BEGIN OF ty_log_out,
```

Comment:
can some of these public types be changed to private?

Reply:
Interfaces may not have private attributes or methods. And the types are used as output of the `ZCL_ABAPGIT_LOG` and handled by `ZCL_ABAPGIT_LOG_VIEWER`.

Reply:
yea, but something seems strange, will move this to separate issue and merge this PR